uaclient/files/notices.py

```diff
@@ -0,0 +1,211 @@
+import os
+from collections import namedtuple
+from enum import Enum
+from typing import List, Tuple
+
+from uaclient import defaults, event_logger, messages, system
+
+event = event_logger.get_event_logger()
+NoticeFileDetails = namedtuple(
+    "NoticeFileDetails", ["order_id", "label", "is_permanent", "message"]
+)
+
+
+class Notice(NoticeFileDetails, Enum):
```

Comment:
I left comments on `is_permanent` values that i think can be changed - the rest look good IMO!